packages-openshift.yaml

```diff
@@ -142,4 +142,16 @@ postprocess:
       rm -f /etc/selinux/targeted/*.LOCK
       # cache, logs, etc...
       rm -rf /var && mkdir /var
+      # All the entries here should instead be part of their respective
+      # packages. But we carry them here for now to maintain compatibility.
+      cat > /usr/lib/tmpfiles.d/openshift.conf << EOF
+    L /opt/cni - - - - ../../usr/lib/opt/cni
+    d /var/lib/cni 0755 root root - -
+    d /var/lib/cni/bin 0755 root root - -
+    d /var/lib/containers 0755 root root - -
+    d /var/lib/openvswitch 0755 root root - -
+    d /var/lib/openvswitch/pki 0755 root root - -
+    d /var/log/openvswitch 0750 openvswitch hugetlbfs - -
+    d /var/lib/unbound 0755 unbound unbound - -
+    EOF
```

Comment:
The indentation looks off here

Reply:
It's actually correct. :) It's a heredoc, and so would include the whitespace in the file we're writing if we indented it.